packages/completer/style/base.css

```diff
@@ -64,6 +64,7 @@
   display: grid;
   grid-template-columns: min-content 1fr min-content;
   position: relative;
+  white-space: nowrap;
```

Comment:
Question to self: should it go to `.jp-Completer-item` or `.jp-Completer-match`?

For context, `jupyterlab-lsp` has more complex support for elision of completions and it sets it on `.jp-Completer-match` ([here](https://github.com/jupyter-lsp/jupyterlab-lsp/blob/main/packages/completion-theme/style/index.css#L38)). I need to check if there was a reason for this in LSP, or if it is a meaningless difference.

Reply:
@mdietz94 would it work for you if this rule was moved to `jp-Completer-item .jp-Completer-match` below?

Reply:
Yeah this still works, moved it in the PR